spec/controllers/user_targets_controller_spec.rb

```diff
@@ -145,6 +145,26 @@
         end
       end
     end
+
+    context 'when there are too many targets created' do
+      let(:new_target) { target_params('title', 50, topic.id, 0, 0) }
+
+      before(:each) do
+        for i in 0..9 do
```

Comment:
Instead of using a for block, you can use the `create_list` method of `FactoryGirl`. Check http://www.rubydoc.info/github/thoughtbot/factory_girl/FactoryGirl/Syntax/Methods